statwidget.py

```diff
@@ -727,15 +729,16 @@ def sortByComponents(self):
             # This causes an error when it runs
 #             sel_mod.select(index, QItemSelectionModel.Select|QItemSelectionModel.Rows)
             self.treeview.setCurrentIndex(index)
-    
+
         self.treeview.setSelectionModel(sel_mod)
-            
-            
-app = QApplication.instance()
-if app is None:
-    app = QApplication(sys.argv)
-else:
-    print('QApplication instance already exists: %s' % str(app))
-ex = StatsGui(dc)
-ex.show()
-sys.exit(app.exec_())
\ No newline at end of file
+
+
+if __name__ == '__main__':
```

Comment:
This makes the following block of code run only when you do ``python statwidget.py``, not if you import the file.